sw/device/silicon_creator/manuf/lib/flash_info_fields.h

```diff
@@ -18,6 +20,20 @@ typedef struct flash_info_field {
 } flash_info_field_t;
 
 enum {
+  /**
+   * AST Calibration Data Size - Bank 0, Page 0
+   *
+   * The format of data written to this field is:
+   *
+   * <# of address/data pairs><<address><data>>...<<address><data>>
+   *
+   * where each subfield (including the first length field) is a 32-bit word.
+   */
+  kFlashInfoFieldMaxAstCalibrationDataSizeInBytes =
+      ((AST_REGAL_REG_OFFSET + 4) * 2) + 4,
```

Comment:
Can you we double check that this is indeed 0x98? It's not clear in the ast.hjson file.

Reply:
<img width="341" alt="Screenshot 2023-11-28 at 1 58 45 PM" src="https://github.com/lowRISC/opentitan/assets/5633066/0eb6afd8-8607-4cf1-a2ce-7cb2b19ea468">
confirmed, but I left it as the autogenerated constant so this can get automatically updated in the future if ast ever changes. Is there a reason why we should not do this?